### DISCUSSION TITLE
New to this

### DISCUSSION THREAD

**Discussion by Jvreal4447** [community/community#43174] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I opened my phone and found what looks like coding and some software from github? What is this? Who is this? Is someone stalking me? Cause I feel like this is what it is?

**Comment by radar-blip** [comment]
Hard for us to say without seeing the code.

**Reply by Jvreal4447** [reply]
#
![Screenshot_20230102-233752_OneDrive](https://user-images.githubusercontent.com/121847156/210308839-2f7b8d08-123a-43d0-b245-14861d74602a.jpg)

**Reply by radar-blip** [reply]
Did you send wrong screenshot? I dont see any code, or anything from GitHub on that.
